AggregateMetrics/AggregateMetrics/AzureWebApp/DefaultCounters.cs

```diff
@@ -0,0 +1,122 @@
+﻿using Microsoft.ApplicationInsights.DataContracts;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.AzureWebApp
+{
+    internal class DefaultCounters
+    {
+        private static readonly DefaultCounters instance = new DefaultCounters();
+
+        private DefaultCounters() { this.Initialize(); }
+
+        /// <summary>
+        /// Return the only instance of DefaultCounters.
+        /// </summary>
+        public static DefaultCounters Instance
+        {
+            get
+            {
+                return instance;
+            }
+        }
+
+        Dictionary<FlexiblePerformanceCounterGauge, MetricTelemetry> defaultCounters;
+        Dictionary<FlexiblePerformanceCounterGauge, TimeDependentCounter> timeDependentCounters;
+
+        /// <summary>
+        /// Initializes the dictionaries for the default performance counters.
+        /// </summary>
+        public void Initialize()
+        {
+            defaultCounters = new Dictionary<FlexiblePerformanceCounterGauge, MetricTelemetry>()
+            {
+                { new FlexiblePerformanceCounterGauge("kernelTime"), new MetricTelemetry() },
+                { new FlexiblePerformanceCounterGauge("userTime"), new MetricTelemetry() },
+                { new FlexiblePerformanceCounterGauge("appRequestExecTime"), new MetricTelemetry() },
+                { new FlexiblePerformanceCounterGauge("privateBytes"), new MetricTelemetry() },
+                { new FlexiblePerformanceCounterGauge("requestsInApplicationQueue"), new MetricTelemetry() }
+            };
+
+            timeDependentCounters = new Dictionary<FlexiblePerformanceCounterGauge, TimeDependentCounter>()
+            {
+                { new FlexiblePerformanceCounterGauge("requestsTotal"), new TimeDependentCounter() },
+                { new FlexiblePerformanceCounterGauge("exceptionsThrown"), new TimeDependentCounter() },
+                { new FlexiblePerformanceCounterGauge("readIoBytes"), new TimeDependentCounter() },
+                { new FlexiblePerformanceCounterGauge("writeIoBytes"), new TimeDependentCounter() },
+                { new FlexiblePerformanceCounterGauge("otherIoBytes"), new TimeDependentCounter() }
```

Comment:
Yeah. Something like Gauge class that sum up other gauges. And each of child gauge is taking one of json values. Like:

SumUpGauge(new FlexiblePerformanceCounterGauge("readIoBytes"), new FlexiblePerformanceCounterGauge("writeIoBytes"), new FlexiblePerformanceCounterGauge("otherIoBytes"));